This is to fix up the python scripts that test the various openbabel
executables. They were looking for the executables relative to the 
testing folder, under the assumption they were part of the usual
cmake, make, ctest, suite. But we instead want to test the binaries that 
we just installed somewhere else (eg. the anaconda _test environment).

diff --git a/test/testbabel.py b/test/testbabel.py
index 5def841..ccf6bbc 100644
--- test/testbabel.py
+++ test/testbabel.py
@@ -60,6 +60,12 @@ def executable(name):
     if sys.platform == "win32":
         suffix = ".exe"
         folder = os.path.join(folder, "Release")
+    # For anaconda testing purposes, the correct binary 
+    # that we wish to test ought to be on the path,
+    from distutils import spawn
+    path_to_exe = spawn.find_executable(name)
+    return path_to_exe
+    # instead of assuming it's somewhere relative to the testing folder:
     return os.path.join("..", folder, name + suffix)
 
 def log(text):